src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
```

Comment:
This is syntactically correct, however `return` statement can be omitted: `const Footer = () => (<div>...</div>)`